src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx

```diff
@@ -177,7 +180,7 @@ const AddLiquidity = ({ currencyA, currencyB, onComplete, onAddToFarm, type }: A
           ' ' +
           currencies[Field.CURRENCY_B]?.symbol
       })
-
+      queryClient.refetchQueries(['get-minichef-farms-v2', account])
```

Comment:
lets create utility hook something like 
`const refetchMinichefSubgraph = useRefetchMinichefSubgraph()` 
which just wrap `queryClient.refetchQueries(['get-minichef-farms-v2', account])` so we don't have to repeat everytime.